### ISSUE TITLE
Pet Merging problem

### ISSUE THREAD

**Issue by Milton3450** [Geigerkind/DPSMate#217]
I have checked "Merge Pets" in the general options tab. My own pet is merged with me but other hunters' and locks' pets are seperate (brown color like warriors). Is this a legit bug or am I missing something?

**Comment by DevineLykan** [comment]
Yeah same for me, also my own's pet damage won't merge unless I've done damage. I left my pet to do just do damage and I never appeared till I did dmg.
